edgemesh/cmd/edgemesh.go

```diff
@@ -29,8 +30,15 @@ func main() {
 	control.Init(opts)
 	opt := registry.Options{}
 	registry.DefaultServiceDiscoveryService = edgeregistry.NewServiceDiscovery(opt)
-	loadbalancer.InstallStrategy(loadbalancer.StrategyRandom, func() loadbalancer.Strategy {
-		return &loadbalancer.RandomStrategy{}
+	myStrategy := mconfig.CONFIG.GetConfigurationByKey("mesh.loadbalance.strategy-name").(string)
```

Comment:
No need to modify this file, these are duplicate codes, I will delete them

Reply:
I have modified, please take a look again